i18n/en.json

```diff
@@ -320,6 +320,26 @@
     "translation": "%s updated the channel header to: %s"
   },
   {
+    "id": "api.channel.post_update_channel_purpose_message_and_forget.join_leave.error",
+    "translation": "Failed to post join/leave message %v"
```

Comment:
This existed above with channel header, I thought it was some special case. I'll change both.